sidebar.md

```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 ##Filter Posts
 [MOD POST](https://www.reddit.com/r/AsianBeauty/search?q=+flair:mod-post&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [REVIEW](https://www.reddit.com/r/AsianBeauty/search?q=+flair:review&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[PSA](https://www.reddit.com/r/AsianBeauty/search?q=+flair:psa&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [QUESTION](https://www.reddit.com/r/AsianBeauty/search?q=+flair:question&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[SWATCHES & FOTD](https://www.reddit.com/r/AsianBeauty/search?q=+flair:swatches-fotd&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[HAULS](https://www.reddit.com/r/AsianBeauty/search?q=+flair:hauls&restrict_sr=on&sort=new&t=all&feature=legacy_search)
+[SWATCHES](https://www.reddit.com/r/AsianBeauty/search?q=+flair:swatches&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [DISCUSSION](https://www.reddit.com/r/AsianBeauty/search?q=+flair:discussion&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [BUSINESS](https://www.reddit.com/r/AsianBeauty/search?q=+flair:business&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[FLUFF](https://www.reddit.com/r/AsianBeauty/search?q=+flair:fluff&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[GLAMDAY](https://www.reddit.com/r/AsianBeauty/search?q=+flair:glamday&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [DISCOVERY](https://www.reddit.com/r/AsianBeauty/search?q=+flair:discovery&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[WEEKLY ROUNDUP](https://www.reddit.com/r/AsianBeauty/search?q=+roundup:roundup&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[DEALS](https://www.reddit.com/r/AsianBeauty/search?q=+flair:deals&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [MEETUPS](https://www.reddit.com/r/AsianBeauty/search?q=+meetups:meetups&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [JOURNAL](https://www.reddit.com/r/AsianBeauty/search?q=+flair:journal&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [SHOW ALL](http://goo.gl/9jxY49)
 
 ##Filter-Out Posts
 [QUESTION](http://qu.reddit.com/r/AsianBeauty/#qu)
-[SWATCHES & FOTD](http://sw.reddit.com/r/AsianBeauty/#sw)
+[SWATCHES](http://sw.reddit.com/r/AsianBeauty/#sw)
 [HAULS](http://ha.reddit.com/r/AsianBeauty/#ha)

```

Comment:
Because we're corralling hauls and fluff into specific days? (Look, I'm poking around and learning!)